scalableminds/webknossos
# Only show locked banner if current user is in organization of the annotation

And do not show it when the annotation is from another organization

### URL of deployed dev instance (used for testing):
- https://___.webknossos.xyz

### Steps to test:
- create another organization in a new context browser tab (e.g. private window) and stay logged in. 
- As a logged-in user of the sample_orga create a new annotation and make it public & `othersMayEdit` in the sharing settings.
- Copy the link into the private browser tab / open it with the other user. There shouldn't be a locked banner.
- confirm bug: Checkout master and reload the link as the user from the 2nd orga. A locked banner "locked by anonymous user" 

### Issues:
- fixes #7934

------
(Please delete unneeded items, merge only when none are left open)
